packages/api-v4/src/account/events.ts

```diff
@@ -50,6 +50,10 @@ export const markEventSeen = (eventId: number) =>
  *
  * @param eventId { number } ID of the event to designate as read
  *
+ * @deprecated As of `5/20/2025`, this endpoint is deprecated. It will be sunset on `6/17/2025`.
+ *
+ * If you depend on using `read`, you may be able to use `markEventSeen` and `seen` instead.
+ * Please note that the seen endpoint functions differently and will mark all events up to and including the referenced event-id as "seen" rather than individual events.
```

Comment:
optional nits - using event ID to be consistent with the comment for `markEventSeen` (or eventId to be consistent with the args passed in) and `seen` instead of seen

```suggestion
 * Please note that the `seen` endpoint functions differently and will mark all events up to and including the referenced event ID as "seen" rather than individual events.
```